Pipfile

```diff
@@ -6,7 +6,11 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-python-telegram-bot = "*"
+python-telegram-bot = {extras = ["socks"],version = "*"}
+ephem = "*"
```

Comment:
I believe it's better to explicitly lock the version in `Pipfile` too since this way you would avoid unexpected updates when you refresh your `Pipfile.lock`